blocksocial.php

```diff
@@ -1,215 +1,275 @@
+                    ),
+                    array(
+                        'type' => 'text',
+                        'label' => $this->l('RSS URL'),
+                        'name' => 'blocksocial_rss',
+                        'desc' => $this->l('The RSS feed of your choice (your blog, your store, etc.).'),
+                    ),
+                    array(
+                        'type' => 'text',
+                        'label' => $this->l('YouTube URL'),
+                        'name' => 'blocksocial_youtube',
+                        'desc' => $this->l('Your official YouTube account.'),
+                    ),
+                    array(
+                        'type' => 'text',
+                        'label' => $this->l('Google+ URL:'),
+                        'name' => 'blocksocial_google_plus',
+                        'desc' => $this->l('Your official Google+ page.'),
+                    ),
+                    array(
+                        'type' => 'text',
+                        'label' => $this->l('Pinterest URL:'),
+                        'name' => 'blocksocial_pinterest',
+                        'desc' => $this->l('Your official Pinterest account.'),
+                    ),
+                    array(
+                        'type' => 'text',
+                        'label' => $this->l('Vimeo URL:'),
+                        'name' => 'blocksocial_vimeo',
+                        'desc' => $this->l('Your official Vimeo account.'),
+                    ),
+                    array(
+                        'type' => 'text',
+                        'label' => $this->l('Instagram URL:'),
+                        'name' => 'blocksocial_instagram',
+                        'desc' => $this->l('Your official Instagram account.'),
+                    ),
+                ),
+                'submit' => array(
+                    'title' => $this->l('Save'),
+                )
+            ),
+        );
+
+        $helper = new HelperForm();
+        $helper->show_toolbar = false;
+        $helper->table =  $this->table;
+        $lang = new Language((int)Configuration::get('PS_LANG_DEFAULT'));
+        $helper->default_form_language = $lang->id;
+        $helper->allow_employee_form_lang = Configuration::get('PS_BO_ALLOW_EMPLOYEE_FORM_LANG') ? Configuration::get('PS_BO_ALLOW_EMPLOYEE_FORM_LANG') : 0;
+        $helper->identifier = $this->identifier;
+        $helper->submit_action = 'submitModule';
+        $helper->currentIndex = $this->context->link->getAdminLink('AdminModules', false).'&configure='.$this->name.'&tab_module='.$this->tab.'&module_name='.$this->name;
+        $helper->token = Tools::getAdminTokenLite('AdminModules');
+        $helper->tpl_vars = array(
+            'fields_value' => $this->getConfigFieldsValues(),
+            'languages' => $this->context->controller->getLanguages(),
+            'id_language' => $this->context->language->id
+        );
```

Comment:
I believe some lines are useless, maybe we could get rid of them:
```php
$lang = new Language((int)Configuration::get('PS_LANG_DEFAULT'));
$helper->default_form_language = $lang->id;
$helper->allow_employee_form_lang = Configuration::get('PS_BO_ALLOW_EMPLOYEE_FORM_LANG') ? Configuration::get('PS_BO_ALLOW_EMPLOYEE_FORM_LANG') : 0;
...
'languages' => $this->context->controller->getLanguages(),
'id_language' => $this->context->language->id
```

This probably need to be tested tho.